introduction.md

```diff
@@ -1,3 +1,9 @@
 This is an introduction...
 
 My favourite path is the JavaScript Path
+
+deadly skills are:
+*JavaScript
+*Git
+*Java
+*Algorithms
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?